pyproject.toml

```diff
@@ -80,7 +80,15 @@ exclude = [
 relative_files = true
 concurrency = ["gevent"]
 source = ["."]
-omit = ["tests/**", "docs/**", "lib/**", "snap/**", "build/**", "setup.py"]
+omit = [
+    "tests/**",
+    "docs/**",
+    "lib/**",
+    "snap/**",
+    "build/**",
+    "setup.py",
+    "cou/steps/hypervisor.py",
```

Comment:
This is for dev branch and without this CI will failes, which we do not want to do. 
We will do PR from dev branch to main, and if we missing thinks like this <==> bad review, if you have a better idea how we can not break CI and not putting it here, I'll go with it.

Reply:
I think it's good to add a comment like `TODO` to remove it later.

Reply:
Yea I suggested this because there were no CI for this branch. But now that you've updated `check.yaml`, I think we can keep it as is. Would be nice to track it somehow, but I don't have better ideas.

Reply:
TODO will cause further issues with linting. Yes, because of your comment, I realized that CI is not working.